packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -46,6 +48,7 @@ export const PersonalDetailsForm = observer(({ history }: { history: BrowserHist
     const [is_btn_loading, setIsBtnLoading] = React.useState(false);
     const [is_submit_success, setIsSubmitSuccess] = React.useState(false);
 
+    const { showPhoneVerificationPage, setShowPhoneVerificationPage } = usePhoneVerification();
```

Comment:
since you're still coding in the legacy codebase, would be best to follow the same naming conventions.

Reply:
Will change the naming conventions here 🙏